extension/README.md

```diff
@@ -296,6 +298,74 @@ Keyboard navigation:
 
 ![Edit Xliff Document](images/gifs/XliffEditorUsage.gif)
 
+### NAB: Export Translations to .tsv
+
+Exports transunits from a selected XLF file as tab separated values.
+
+Steps:
+1. Run `NAB: Export Translations from .tsv` from command palette.
+2. Select which XLF files to update, one or more needs to be selected.
+3. File is exported.
+
+- Export path:
+  - If a path is set for setting `NAB.XliffTSVExportPath` that path is used. Else it defaults to the Translation folder.
+- Exported file name: `[App Name].[Language].csv`
+- Separator: `\t` (TAB)
+- EOL: `\r\n` (CRLF)
+- Column order:
+    1. Trans unit Id *
+    2. Source *
+    3. Target *
+    4. Developer Note
+    5. Max Length
+    6. Comment (this will be empty, used for commenting changes in Excel later on)
+    7. Xliff Generator Note
+    8. NAB AL Tool Note
+    9. State
+    10. State Qualifier
+
+\* required column.
+
+**Example**
+```xml
+<xliff>
+<file datatype="xml" source-language="en-US" target-language="sv-SE" original="AlTestApp">
+    <body>
+      <group id="body">
+        <trans-unit id="Table 2328808854 - NamedType 12557645" size-unit="char" maxwidth="50" translate="yes" xml:space="preserve">
+          <source>This is a test</source>
+          <target state="final" state-qualifier="exact-match">Detta är ett test</target>
+          <note from="Developer" annotates="general" priority="2">Some kind of Dev note</note>
+          <note from="Xliff Generator" annotates="general" priority="3">Table MyTable - NamedType TestErr</note>
+          <note from="NAB AL Tool Refresh Xlf" annotates="general" priority="3">Source has been modified.</note>
+        </trans-unit>
+        <trans-unit id="Page 2931038265 - NamedType 12557645" size-unit="char" translate="yes" xml:space="preserve">
+          <source>Cool</source>
+          <target>Sval</target>
+          <note from="Developer" annotates="general" priority="2"/>
+          <note from="Xliff Generator" annotates="general" priority="3">Page MyPage - NamedType TestErr</note>
+        </trans-unit>
+      </group>
+    </body>
+  </file>
+</xliff>
+```
+is exported to
+```tsv
+Id	Source	Target	Developer	Max Length	Comment	Xliff Generator Note	NAB AL Tool Refresh Xlf	State	State Qualifier
+Table 2328808854 - NamedType 12557645	This is a test	Detta är ett test	Some kind of Dev note	50		Table MyTable - NamedType TestErr	Source has been modified.	final	exact-match
+Page 2931038265 - NamedType 12557645	Cool	Sval				Page MyPage - NamedType TestErr			
+```
+ExportTranslationsTSV.gif
```

Comment:
I prepared som gifs but since it's a high probability that I have to redo them I left them out. And was kind of hoping that we might skip them for this one if the description is could enough. What do you think?

Reply:
Skip the image! 👍